docs/en/ingest-management/tab-widgets/add-fleet-server/content.asciidoc

```diff
@@ -25,86 +25,63 @@ image::images/integrations-server-hosted-container.png[Hosted {integrations-serv
 
 // tag::self-managed[]
 
-To deploy a self-managed {fleet-server}, install an {agent} and enroll it in an
-agent policy containing the {fleet-server} integration.
+To deploy a self-managed {fleet-server}, you install an {agent} and enroll it in
+an agent policy containing the {fleet-server} integration.
 
 NOTE: You can install only a single {agent} per host, which means you cannot run
 {fleet-server} and another {agent} on the same host unless you deploy a
 containerized {fleet-server}.
 
-. Log in to {kib} and go to *Management > {fleet} > Settings*. For more
-information about these settings, see
-{fleet-guide}/fleet-settings.html[{fleet} settings].
+. In {kib}, go to **Management > {fleet} > Settings**. For more information
+about these settings, see {fleet-guide}/fleet-settings.html[{fleet} settings].
 // lint ignore fleet-server
-. Under *Fleet Server hosts*, click *Edit hosts* and specify one or more host
+. Under **Fleet Server hosts**, click **Edit hosts** and specify one or more host
 URLs your {agent}s will use to connect to {fleet-server}. For example,
 `https://192.0.2.1:8220`, where `192.0.2.1` is the host IP where you will
 install {fleet-server}. Save and apply your settings.
 
-. In the *{es} hosts* field, specify the {es} URLs where {agent}s will send data.
+. In the **{es} hosts** field, specify the {es} URLs where {agent}s will send data.
 For example, `https://192.0.2.0:9200`. Skip this step if you've started the
 {stack} with security enabled (you cannot change this setting because it's
 managed outside of {fleet}).
 
 . Save and apply the settings.
 
-. Click the *Agents* tab and follow the in-product instructions to add a
+. Click the **Agents** tab and follow the in-product instructions to add a
 {fleet} server:
 +
 [role="screenshot"]
 image::images/add-fleet-server.png[In-product instructions for adding a {fleet-server}]
 
-*Notes:*
+**Notes:**
 
-* Make sure you download an x64 architecture installation package.
-
-* For the agent policy, choose one with a {fleet-server} integration, or click
-*Create policy* to create one now. Alternatively you can
+* Choose **Quick Start** if you want {fleet} to generate a
```

Comment:
What else do we need to say here?

Reply:
Not sure what else needs to be said, this is what quick start does at a high level:
- creates a policy with a preset name "Fleet Server Policy", with the fleet server and system integration added for monitoring the agent. 
- uses "quick start" security (which is not recommended for production use as in screenshot below)
- generates an enrollment token